include/anvill/MCToIRLifter.h

```diff
@@ -110,21 +110,38 @@ class MCToIRLifter {
   void VisitIndirectJump(const remill::Instruction &inst,
                          remill::Instruction *delayed_inst,
                          llvm::BasicBlock *block);
+  void VisitConditionalIndirectJump(const remill::Instruction &inst,
+                           remill::Instruction *delayed_inst,
+                           llvm::BasicBlock *block);
+
   void VisitFunctionReturn(const remill::Instruction &inst,
                            remill::Instruction *delayed_inst,
                            llvm::BasicBlock *block);
 
+  void VisitConditionalFunctionReturn(const remill::Instruction &inst,
```

Comment:
So I think I understand what a few of these should do, but the name, but what is a conditional function return?

Reply:
It is a function return that is conditional on the condition codes :-P In aarch32, function calls/returns can be conditional.